app/notification.go

```diff
@@ -691,7 +715,7 @@ type postNotification struct {
 func (n *postNotification) GetChannelName(userNameFormat string, excludeId string) string {
 	switch n.channel.Type {
 	case model.CHANNEL_DIRECT:
-		return fmt.Sprintf("@%s", n.sender.GetDisplayName(userNameFormat))
+		return n.sender.GetDisplayName(userNameFormat)
```

Comment:
Should we leave this as it is for supporting older mobile apps? As it is, I assume the channel name won't be displayed on updated apps.

Reply:
Nope, this was changed in #10298 but they missed this one